File: AllSpice.client/src/models/Recipe.js
function convertInstructionsToSteps(instructions) {
  const steps = instructions.split(" + ");
  return steps;
}

export class Recipe {
  constructor(data) {
    this.id = data.id
    this.title = data.title
    this.instructions = data.instructions
    this.img = data.img
    this.category = data.category
    this.creatorId = data.creatorId
    this.creator = data.creator
    this.description = data.description
    this.steps = convertInstructionsToSteps(data.instructions)

  }
}

File: AllSpice/Models/Recipe.cs
namespace AllSpice.Models;

public class Recipe
{
  public int Id { get; set; }
  public string Title { get; set; }
  public string Description { get; set; }
  public string Instructions { get; set; }
  public string Img { get; set; }
  public string Category { get; set; }
  public string CreatorId { get; set; }
  public Account Creator { get; set; }
  
}
public class FavoriteRecipe : Recipe
{
    public int FavoriteId { get; set; }
}
 public class CollectionRecipe : Recipe // NOTE this is a view model
  {
    public int CollectionId { get; set; }
  }
